.travis.yml

```diff
@@ -1,5 +1,5 @@
 os: osx
-osx_image: xcode10
+osx_image: xcode10.2
```

Comment:
Probably should bump the simulator versions as well in here.

Reply:
Simulators have been bumped up.